controllers/admin/AdminPsMboModuleController.php

```diff
@@ -240,11 +240,16 @@ public function ajaxProcessGetTabModulesList()
                 'host_mode' => defined('_PS_HOST_MODE_') ? 1 : 0,
                 'from' => 'tab'
             ));
-            $this->smartyOutputContent($this->module->template_dir . '/admin-end-content.tpl');
+            $this->smartyOutputContent($this->module->template_dir . '/include/admin-end-content-footer.tpl');
         }
 
         exit;
     }
+    
+    public function ajaxProcessFetchModules() {
+        $result = $this->module->fetchModules(Tools::getValue('controller_page'));
+        die($result);
```

Comment:
prefer `$this->ajaxRender(...)` instead of dying, you will allow the post process hooks to be run.

Reply:
I can not access this function :-(